src/components/drawer/Drawer.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import './Drawer.scss';
+import {
+  CategoryFilter,
+  RatingFilter,
+  PriceSortFilter,
+  PriceRangeFilter,
+} from '../product-filter';
+
+export const Drawer = () => {
```

Comment:
Instead of naming it Drawer, rename it to FilterDrawer or DrawerFilter so it is easy to understand that it contains filters. Otherwise Drawers can be many thing.